### ISSUE TITLE
Selecting schema

### ISSUE THREAD

**Issue by je111ena** [tortoise/tortoise-orm#171]
To include in db config (or elsewhere) which existing schema should be used.
Currently, there is no option that enables specifying schema.

For example: 

```
{
    'connections': {
        # Dict format for connection
        'default': {
            'engine': 'tortoise.backends.asyncpg',
            'credentials': {
                'host': 'localhost',
                'port': '5432',
                'user': 'tortoise',
                'password': 'qwerty123',
                'database': 'test',
```
                'schema': 'test_schema'
```
            }
        },
        # Using a DB_URL string
        'default': 'postgres://postgres:qwerty123@localhost:5432/events'
    },
    'apps': {
        'models': {
            'models': ['__main__'],
            # If no default_connection specified, defaults to 'default'
            'default_connection': 'default',
        }
    }
}

```

**Comment by grigi** [comment]
I looked at asyncpg, and it doesn't have a way to set the schema during a connection.

Reading up on PostgreSQL docs: https://www.postgresql.org/docs/11/ddl-schemas.html
It seems that your current schema can be selected like so:
```sql
SET search_path TO myschema;
```
Which would prefix all unqualified table names with `myschema.`.
Which would probably be OK as a first version of schema support?

Is this what you are thinking of?

**Comment by je111ena** [comment]
Yes, that's what I thought.

**Comment by grigi** [comment]
Possibly implemented as a special parameter `schema` that can be specified as a query-string parameter. If it exists, we run `Set search_path TO {schema}`.
Now how does one test that this does what we expect? :thinking:

**Comment by je111ena** [comment]
Maybe by creating several schemes in the DB, and checking if the query executes over the right one?

**Comment by grigi** [comment]
Yeah, I don't see any other way. This does mean we will have to introduce DB-specific setup in the tests though. So far have only used DB's in default config for testing.

Possibly can do so just by manual testing for now, and add a TODO that we want to automate it at a later stage (When we have more such cases)

**Comment by grigi** [comment]
@je111ena Could you please see if this resolves your issue?

**Comment by grigi** [comment]
Released v0.13.1

**Comment by karan-matalia** [comment]
Is it possible to select the schema during query time?
Something like MyModel.filter(**filters).using_schema('my_schema_name')
To query like select * from my_schema_name.MyModel where ...
We have a use case where we will be selecting the schema name for querying dynamically at run time. So we might have to query tables in multiple schemas for different schenarios.
